src/mantine-dates/src/components/DatePickerBase/DatePickerBase.tsx

```diff
@@ -276,14 +280,15 @@ export const DatePickerBase = forwardRef<HTMLInputElement, DatePickerBaseProps>(
         {...wrapperProps}
       >
         <Popover
-          position="bottom-start"
           __staticSelector={__staticSelector}
           withinPortal={withinPortal}
           offset={10}
           opened={dropdownOpened}
           transitionDuration={transitionDuration}
           transition={transition}
           positionDependencies={positionDependencies}
+          middlewares={{ flip: dropdownPosition === 'flip', shift: false }}
+          position={dropdownPosition === 'flip' ? 'bottom' : dropdownPosition}
```

Comment:
With these changes dropdown will be displayed at the bottom/top center while previously it was displayed at bottom/top start (`bottom-start` position)

Reply:
Should be fix, just added bottom-start and top-start, should we add others FloatingPosition ?

Reply:
Should be fixed, not sure if we add others FloatingPositions